tabcmd/commands/datasources_and_workbooks/datasources_and_workbooks_command.py

```diff
@@ -111,19 +120,23 @@ def is_truthy(value: str):
         return value.lower() in ["yes", "y", "1", "true"]
 
     @staticmethod
-    def apply_png_options(request_options: TSC.ImageRequestOptions, args, logger):
-        if args.height or args.width:
-            # only applicable for png
-            logger.warn("Height/width arguments not yet implemented in export")
+    def apply_png_options(logger, request_options: TSC.ImageRequestOptions, args):
+        try:
+            if args.height or args.width:
+                # only applicable for png
+                logger.warn("Height/width arguments not yet implemented in export")
+        except AttributeError as ae:
+            logger.debug("Unexpected: no user or default value is present: {}".format(ae.__str__()))
         # Always request high-res images
         request_options.image_resolution = "high"
 
     @staticmethod
     def apply_pdf_options(request_options: TSC.PDFRequestOptions, args, logger):
```

Comment:
Did you want to reorder the arguments on this method too?

Reply:
yes :)